public/scripts/editor/js/fc/path-utils.js

```diff
@@ -0,0 +1,83 @@
+define(function() {
+  var Path = Bramble.Filer.Path;
+  var uuid = require("uuid");
+  var fs = Bramble.getFileSystem();
+  var sh = fs.Shell();
+
+  // Bramble loads all projects via symlinks stored in entries named /.mnt/<uuid>
+  var MOUNT_DIR = "/.mnt";
+
+  function createMountPoint(root, callback) {
+    var mountPoint = Path.join(MOUNT_DIR, uuid.v4());
+
+    sh.mkdirp(MOUNT_DIR, function(err) {
+      if(err && err.code !== "EEXIST") {
```

Comment:
Shouldn't this just be `if(err)`? If we're generating a UUID it shouldn't exist.